src/Controller/Security/PasswordController.php

```diff
@@ -33,7 +35,9 @@ public function __construct(
     #[Route("/reset/{id}", name: "reset", methods: ["GET", "POST"])]
     public function resetAction(LocalAccount $auth, Request $request): Response
     {
-        if (!$this->passwordReset->isPasswordRequestTokenValid($auth, $request->query->get('token'))) {
+        $token = $request->query->get('token');
+        assert(is_string($token));
+        if (!$this->passwordReset->isPasswordRequestTokenValid($auth, $token)) {
```

Comment:
How about simply:
```php
if (!$this->passwordReset->isPasswordRequestTokenValid($auth, (string) $request->query->get('token'))) {
```
or if you feel more security is required
```php
if (!$this->passwordReset->isPasswordRequestTokenValid($auth, $request->query->getAlnum('token'))) {
```

Reply:
I like the `getAlnum` option, especially since it can only return a string